### ISSUE TITLE
Dissociate the TemplatePage logic from REST/Services

### ISSUE THREAD

**Issue by cmoulliard** [halkyonio/primaza-poc#20]

**Comment by aureamunoz** [comment]
Ok. I would prefer for the Rest Resource to keep the`Resource`in the name because these classes providing the REST CRUD operations use to be named SomethingResource. So `ClaimPageResource`would be better IMO.

**Comment by cmoulliard** [comment]
> I would prefer for the Rest Resource to keep the`Resource`in the name because these classes providing the REST CRUD operations use to be named SomethingResource. So `ClaimPageResource`would be better IMO.

I agree with you. From my point of Qute Template class should be used to:
- Access the services 
- Validate Form
- Prepare response
- Populate some values
- etc
So I(we) will see if it is possible to dissociate the REST endpoints from the templates, etc

**Comment by cmoulliard** [comment]
This is why I dont think that we will still use ResourceProperties of Rest Data Panache as many of the REST endpoints include a HTP response which is not JSON based: https://github.com/MaisiKoleni/javadoc-search/blob/40ebc61f930b61974923d8826672f04b6242c616/src/main/java/net/maisikoleni/javadoc/server/html/JavadocSearchPage.java#L37

**Comment by aureamunoz** [comment]
I think we can close this one in favor of #26. A [PR](https://github.com/halkyonio/primaza-poc/pull/33) has been created for refactoring. WDYT @cmoulliard ?
